ui-particles-angular/projects/ui-particles-angular/src/scss/component/gio-table-light/gio-table-light.scss

```diff
@@ -24,15 +24,15 @@ $typography: map.get(theme.$mat-theme, typography);
 
 @mixin theme() {
   table.gio-table-light {
-    border-collapse: collapse;
+    border: 1px solid map.get(palettes.$mat-dove-palette, darker10);
```

Comment:
I was not sure about this change globally. Do we need to keep it like it is actually and create another class or update the existing one 🤔 ?